MangaRipper/Forms/FormMain.cs

```diff
@@ -21,6 +22,26 @@ public partial class FormMain : Form
         private BindingList<DownloadChapterTask> _downloadQueue;
         private readonly ApplicationConfiguration _appConf = new ApplicationConfiguration();
 
+        private string SaveDestination
+        {
+            get
+            {
+                if (rdSeriesDestination.Checked)
+                    return lbSeriesDestination.Text;
```

Comment:
Best practice suggest to wrap the lines in "if" "else" statement into brackets.
In my practice I have some situation, when we receive a serious bug for not following it.

b.t.w. I recommend to use http://www.sonarlint.org/visualstudio/ for VS.